modules/theme-tools/compat/twentytwentyone.php

```diff
@@ -10,6 +10,21 @@
  * Add Jetpack extra functionality to Twenty Twenty One.
  */
 function twentytwentyone_jetpack_setup() {
+
+	/**
+	 * Add theme support for Infinite Scroll.
+	 */
+	add_theme_support(
+		'infinite-scroll',
+		array(
+			'type'      => 'click',
+			'container' => 'main',
+			'wrapper'   => false,
+			'render'    => 'twentytwentyone_infinite_scroll_render',
+			'footer'    => 'main',
```

Comment:
I think we may need to take the footer widgets into account:

```suggestion
			'type'           => 'click',
			'container'      => 'main',
			'wrapper'        => false,
			'render'         => 'twentytwentyone_infinite_scroll_render',
			'footer'         => 'main',
			'footer_widgets' => array( 'sidebar-1' ),
```

Reply:
I was debating this when I was working on this @jeherve I am not sure. I think that option is specifically for the scrolling reload option, and since we have it set to 'click' it shouldn't matter? https://jetpack.com/support/infinite-scroll/
"The footer_widgets argument accepts a variety of values and indicates whether or not a theme has footer widgets. If footer widgets are present, the scroll type will be set to click so the widgets are accessible."

From what I found we added the footer_widget option on twentytwenty, but twentynineteen did not have it:
https://github.com/WordPress/twentynineteen/blob/master/functions.php#L184
https://github.com/Automattic/jetpack/blob/master/modules/theme-tools/compat/twentynineteen.php#L12

And if we do take the widgets into account should we account for the footer menu also?

Reply:
> I think that option is specifically for the scrolling reload option, and since we have it set to 'click' it shouldn't matter?

That's a good point. Since it's a click it should be okay 👍